### PR TITLE
Add test to check that duplicate entries are ignored

### PR DESCRIPTION
**Pull request by 0xN3utr0n**
This PR solves [#325 ](https://github.com/wazuh/wazuh-qa/issues/325).
It tries cases like so:           
> \<directories realtime="yes">/home/user\</directories> 
> \<directories whodata="yes">/home/user\</directories>  

AND

>\<directories realtime="yes">/home/user,/home/user\</directories>

In both scenarios only one event should be generated.

**Linux**
```
python3 -m pytest -vv test_fim/test_ambiguous_confs/test_duplicate_entries.py 
========================================================= test session starts =========================================================
platform linux -- Python 3.6.8, pytest-5.3.2, py-1.8.0, pluggy-0.13.1 -- /bin/python3
cachedir: .pytest_cache
rootdir: /home/vagrant/wazuh-qa/test_wazuh, inifile: pytest.ini
collected 9 items                                                                                                                     

test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration0-tags_to_apply0] PASSED       [ 11%]
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration1-tags_to_apply0] PASSED       [ 22%]
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration2-tags_to_apply0] PASSED       [ 33%]
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration3-tags_to_apply0] PASSED       [ 44%]
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration4-tags_to_apply0] PASSED       [ 55%]
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration5-tags_to_apply0] PASSED       [ 66%]
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration6-tags_to_apply0] PASSED       [ 77%]
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration7-tags_to_apply0] PASSED       [ 88%]
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration8-tags_to_apply0] PASSED       [100%]

============================================== 9 passed in 140639.74s (1 day, 15:03:59) ===============================================
```

**Windows**
```
========================================================= test session starts ==========================================================
platform win32 -- Python 3.7.3, pytest-5.1.2, py-1.8.0, pluggy-0.13.0 -- C:\Users\jmv74211\AppData\Local\Programs\Python\Python37-32\pyth
on.exe
cachedir: .pytest_cache
rootdir: C:\Users\jmv74211\Desktop\wazuh-qa\test_wazuh, inifile: pytest.ini
collected 9 items

test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration0-tags_to_apply0] PASSED  [ 11%]                                                                                                           
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration1-tags_to_apply0] PASSED  [ 22%]                                                                                                           
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration2-tags_to_apply0] PASSED  [ 33%]                                                                                                            
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration3-tags_to_apply0] PASSED  [ 44%]                                                                                                            
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration4-tags_to_apply0] PASSED  [ 55%]                                                                                                             
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration5-tags_to_apply0] PASSED  [ 66%]                                                                                                            
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration6-tags_to_apply0] PASSED  [ 77%]                                                                                                             
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration7-tags_to_apply0] PASSED  [ 88%]                                                                                                            
test_fim/test_ambiguous_confs/test_duplicate_entries.py::test_duplicate_entries[get_configuration8-tags_to_apply0] PASSED  [100%]                                                                                                            

================================================================================================= 9 passed in 140683.67s (1 day, 15:04:43) ==================================================================================================
```

### CODE REVIEW

File: test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2015-2019, Wazuh Inc.
```
Comment (crd1985) [inline test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py]: This module should be place under test_fim/test_ambiguous_confs/test_duplicate_entries.py

File: test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+from wazuh_testing.fim import (CHECK_ALL, LOG_FILE_PATH, check_time_travel, callback_detect_event, 
+    create_file, REGULAR, detect_initial_scan, delete_file)
+from wazuh_testing.tools import (FileMonitor, check_apply_test, load_wazuh_configurations, 
+    restart_wazuh_with_new_conf, set_section_wazuh_conf)
+
+# variables
+
+if sys.platform == 'win32':
+    test_directories = [os.path.join('c:', os.sep, 'testdir1'), os.path.join('c:', os.sep, 'testdir1')]
+    directory_str = "c:\\testdir1,c:\\testdir2,c:\\noexists"
```
Comment (crd1985) [inline test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py]: Use the join function

File: test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+import pytest
+
+from wazuh_testing.fim import (CHECK_ALL, LOG_FILE_PATH, check_time_travel, callback_detect_event, 
+    create_file, REGULAR, detect_initial_scan, delete_file)
+from wazuh_testing.tools import (FileMonitor, check_apply_test, load_wazuh_configurations, 
+    restart_wazuh_with_new_conf, set_section_wazuh_conf)
+
+# variables
+
+if sys.platform == 'win32':
```
Comment (crd1985) [inline test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py]: That platform distinction is not necessary here. Use tools.PREFIX instead.

File: test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+
+def mode_format(mode):
+    if mode is not 'scheduled':
+        return {mode:'yes'}
+    else:
+        return ''
+
+
+def change_conf(mode=None, mode2=None):
```
Comment (crd1985) [inline test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py]: I think this logic is already implemented in the wazuh_testing package.
